kubernetes_test.go

```diff
@@ -87,15 +87,15 @@ func TestKubernetesClusters_ListClusters(t *testing.T) {
 							ID:        "deadbeef-dead-beef-dead-deadbeefb4b1",
 							Name:      "worker-393",
 							Status:    &KubernetesNodeStatus{State: "running"},
-							DropletID: "droplet-3",
+							DropletID: 11113,
```

Comment:
In the Kubernetes response from the API, `droplet_id` is actually a string.

https://docs.digitalocean.com/reference/api/api-reference/#operation/get_kubernetes_cluster